camelot/view/action_runner.py

```diff
@@ -102,6 +102,7 @@ def __init__(
         self.mode = mode
         self.started_at = time.time()
         self.steps = []
+        self.server = None
```

Comment:
Deze variabele met thread/process id gaan we dan in de toekomst gebruiken om te voorkomen dat er berichten bij een foute thread/process terechtkomen? Of is het enkel de boedeloing dat we deze loggen?

Reply:
in deze faze idd logging

maar we zouden dat 'ooit' ook kunnen gebruiken om te verhinderen dat je bvb een response van een action step naar een ander process stuurt dan dewelke vanwaar de action step komt